test/sh_spec.py

```diff
@@ -1395,9 +1395,10 @@ def main(argv):
   # spec/smoke.test.sh -> smoke
   test_name = os.path.basename(test_file).split('.')[0]
 
-  allowed = opts.oils_failures_allowed
+  multiplier = 2 if opts.oils_cpp_bin_dir else 1
+  allowed = opts.oils_failures_allowed * multiplier
   all_count = stats.Get('num_failed')
-  oils_count = stats.Get('oils_num_failed')
+  oils_count = stats.Get('oils_num_failed') * multiplier
```

Comment:
I think we should drop the multipler here, and then say `%d oils python failures`

Because Python and C++ can fail in different ways -- they won't always fail at the same time!

(the alternative is to accurately track osh-cpp and ysh-cpp failures in a separate count, and print both)

Reply:
Even if not perfect this just allows to prevent FATAL while executing. Most of tests, so spend less time to count am ually ;)
I agree there should be a perfect way to do, but we can do it later, iteratively ?